api/src/resources/projects.py

```diff
@@ -73,3 +77,53 @@ def get(self, id):
 
 
 api.add_resource(ProjectLocationsAPI, '/projects/<int:id>/locations', endpoint='project_locations')
+
+
+class UploadAPI(Resource):  
+    def __init__(self):
+        super().__init__()
+        self.parser = reqparse.RequestParser()  # for input validation
+        self.parser.add_argument('file')   # TODO: where should this add be?
```

Comment:
Can you elaborate on the what that TODO means? Is it something that you already completed, or something that should still be done?